api/src/database/migrations/20250613A-add-project-id.ts

```diff
@@ -0,0 +1,26 @@
+import type { Knex } from 'knex';
+import { v7 as uuid } from 'uuid';
+
+export async function up(knex: Knex): Promise<void> {
+	await knex.schema.alterTable('directus_settings', (table) => {
+		table.uuid('project_id');
+	});
+
+	const existing = await knex('directus_settings').select('id').first();
+
+	if (existing) {
+		await knex('directus_settings').update({
+			project_id: uuid(),
```

Comment:
We should get the timestamp of the first database migration

```
const timestamp = await knex('directus_migrations').select('timestamp').first();
``` 

and use that when creating the uuid like 

```
uuid({ msecs: 12345 }) //  RFC "timestamp" field (Number of milliseconds, unix epoch)
```